### PR TITLE
fix: Forgot screen Screen Full Size

### LINKED ISSUE #606: Make Check your email (Forgot password) screen to full screen
Make it similar to Eventbrite. 

<p align="center">
 
 
</p>

### PR DESCRIPTION
**Pull request by shiv07tiwari**
Fixes #606 

Changes:ActionBar and Bottom Navigation bar hidden

Screenshots for the change:
image

### CODE REVIEW

File: app/src/main/java/org/fossasia/openevent/general/auth/LoginFragment.kt [hunk 1]
```diff
@@ -132,11 +133,19 @@ class LoginFragment : Fragment() {
...
+                Utils.navAnimVisible(thisActivity.navigationAuth, context)
+            }
             rootView.loginLayout.visibility = View.VISIBLE
         }
 
         rootView.forgotPassword.setOnClickListener {
             hideSoftKeyboard(context, rootView)
+            if (thisActivity is AppCompatActivity) {
+                thisActivity.supportActionBar?.hide()
+                Utils.navAnimGone(thisActivity.navigationAuth, context)
```
Comment (iamareebjamal) [inline app/src/main/java/org/fossasia/openevent/general/auth/LoginFragment.kt]: Use requireContext()

File: app/src/main/java/org/fossasia/openevent/general/auth/LoginFragment.kt [hunk 2]
```diff
@@ -132,11 +133,19 @@ class LoginFragment : Fragment() {
 
         rootView.tick.setOnClickListener {
             rootView.sentEmailLayout.visibility = View.GONE
+            if (thisActivity is AppCompatActivity) {
+                thisActivity.supportActionBar?.show()
+                Utils.navAnimVisible(thisActivity.navigationAuth, context)
```
Comment (iamareebjamal) [inline app/src/main/java/org/fossasia/openevent/general/auth/LoginFragment.kt]: Same here

**Comment by shiv07tiwari** [comment]
@iamareebjamal Made the changes.

**Comment by shiv07tiwari** [comment]
@nikit19 Edited the title as per semantic PR

File: app/src/main/java/org/fossasia/openevent/general/auth/LoginFragment.kt [hunk 3]
```diff
@@ -14,6 +14,7 @@ import androidx.fragment.app.Fragment
 import androidx.lifecycle.Observer
 import androidx.navigation.Navigation.findNavController
 import com.google.android.material.snackbar.Snackbar
+import kotlinx.android.synthetic.main.activity_main.*
```
Comment (GOVINDDIXIT) [inline app/src/main/java/org/fossasia/openevent/general/auth/LoginFragment.kt]: avoid wildcard imports

**Comment by shiv07tiwari** [comment]
@nikit19 @GOVINDDIXIT  Removed the wildcard imports.
Also i found a logical error in code (The ActionBar and BottomNavigationBar used to hide even when invalid email address is entered.) Hence, fixed that as well.

**Comment by shiv07tiwari** [comment]
videotogif_2019 02 03_19 04 53
@liveHarshit

**Comment by liveHarshit** [comment]
PR title is wrong, it's frogot password screen. And learn git well, every problem has a solution.

**Comment by shiv07tiwari** [comment]
@liveHarshit I am really sorry about the git issue, I have the knowledge of git but this time I messed up the commands. Will not happen in future for sure.

**Comment by shiv07tiwari** [comment]
@iamareebjamal @nikit19 Kindly review this PR.

**Comment by liveHarshit** [comment]
Please review this PR.

**Pull request merged**